search-api/migrations/versions/d5ad68999347_.py

```diff
@@ -0,0 +1,56 @@
+"""empty message
+
+Revision ID: d5ad68999347
+Revises: 
+Create Date: 2021-09-29 20:38:32.148808
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'd5ad68999347'
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('global_dimension',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('name', sa.String(), nullable=True),
+    sa.Column('email', sa.String(), nullable=True),
+    sa.Column('salary', sa.Numeric(), nullable=True),
+    sa.Column('references', sa.String(), nullable=True),
+    sa.PrimaryKeyConstraint('id')
+    )
+    op.create_table('global_dimension_value',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('datasetId', sa.Integer(), nullable=True),
+    sa.PrimaryKeyConstraint('id')
+    )
+    op.create_table('otherthing',
```

Comment:
I think these are just experimental tables, we should remove these before committing.

Reply:
For reference purpose I've commented them.